src/templates/Article.jsx

```diff
@@ -10,13 +10,21 @@ const propTypes = {
 
 function Article(props) {
   const article = props.data.markdownRemark;
-  const { html, frontmatter: { title } } = article;
+  const {
+    html,
+    fields: {
+      slug
+    },
+    frontmatter: {
+      title
+    }
+  } = article;
   return (
     <div>
       <Helmet>
         <title>{ `${title} | freeCodeCamp Guide` }</title>
         <link
-          href={ `https://freecodecamp.org/guide${props.location.pathname}` }
```

Comment:
This failed on `build`

Reply:
@Bouncey OK - I will wait to QA this until I get your go-ahead.

Reply:
Oh, no it was just a note as to why it changed :smile: 

I have rebased this so it should be good to go